packages/react-slider/src/utils/renderMarks.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { mergeClasses } from '@fluentui/react-make-styles';
+
+// The mark related classNames are needed since they are used in a JSX element that is dynamically generated.
+const markContainerClassName = 'ms-Slider-markItemContainer';
+const firstMarkClassName = 'ms-Slider-firstMark';
+const lastMarkClassName = 'ms-Slider-lastMark';
+export const markClassName = 'ms-Slider-mark';
+
+/**
+ * Renders the marks
+ *
+ * The marks percentage position relative to their individual positions.
+ * @param markValues
+ */
+export const renderMarks = (markValues: number[]) => {
+  const marksChildren: JSX.Element[] = [];
```

Comment:
nit, it's cleaner to just use `map`. `return marksValues.map((value, i) => ...)`